src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandler.java

```diff
@@ -313,6 +313,11 @@ public Mono<RequestAuthorizationResponseDto> handle(TransactionRequestAuthorizat
                                     );
 
                                     return transactionEventStoreRepository.save(authorizationEvent)
+                                            .doOnNext(e -> {
+                                                String authorizationRequestId = e.getData().getAuthorizationRequestId();
+                                                String transactionId = t.getTransactionId().value();
+                                                log.info("Logging event store saved authorization event for transactionId: [{}] and authorizationRequestId: [{}]", transactionId, authorizationRequestId);
```

Comment:
```suggestion
                                                log.info("Saved authorization requested event for transactionId: [{}] and authorizationRequestId: [{}]", transactionId, authorizationRequestId);
```
🚲 
just a log rewording

Reply:
you can use https://github.com/pagopa/pagopa-ecommerce-commons/blob/c153b65f8e8d75ef44df6d794123dbae2b3b5488/src/main/java/it/pagopa/ecommerce/commons/domain/v2/TransactionEventCode.java#L16

Reply:
Based on the @pietro-tota base log, I've also added the related event name as suggested by @infantesimone